src/main/java/crazypants/enderio/config/Config.java

```diff
@@ -1199,6 +1215,13 @@ public static void processConfig(Configuration config) {
                 sectionStaff.name,
                 travelStaffBlinkBlackList,
                 "Lists the blocks that cannot be teleported through in the form 'modID:blockName'");
+        travelStaffBaublesType = config.get(
+                sectionStaff.name,
+                "travelStaffBaublesType",
+                travelStaffBaublesType,
+                "The BaublesType the Travel Staff should be, 'AMULET', 'RING' or 'BELT' (requires Baubles to be installed and travelStaffAllowInBaublesSlot to be on)")
```

Comment:
GTNH version of Baubles has [yet another type](https://github.com/GTNewHorizons/Baubles/blob/master/src/main/java/baubles/api/BaubleType.java#L7) which should be supported

Reply:
This is copied directly from the Magnet item's bauble config options. I was simply trying to keep convention.
As on the other one I'll add this option to both shortly.

Reply:
Done in latest commit.